storage/innobase/xtrabackup/test/t/pxb-2980-smart_memory_parameter.sh

```diff
@@ -0,0 +1,35 @@
+# PXB-2980 - Add a parameter to disable memory estimation during backup
```

Comment:
Can we fail this test if the version is 8.0.34. so it reminds us to start a discussion about making this feature GA.

Reply:
Sure. Is .34 a reasonable release?
@satya-bodapati ^^

Reply:
Lets see. We can target that release. If there is a chance to implement page-tracking-based estimation before that, it would be good.

Reply:
Done

Reply:
please avoid bug number as part of file. estimate_memory_param.sh?

Reply:
and mention the bug number inside
PXB-2980 : Add a parameter to disabe..

Reply:
@altmannmarcelo sorry for the last minute request